src/rgw/rgw_sal.cc

```diff
@@ -119,35 +119,6 @@ rgw::sal::Store* StoreManager::init_storage_provider(const DoutPrefixProvider* d
     }
     ((rgw::sal::MotrStore *)store)->init_metadata_cache(dpp, cct);
 
-    /* XXX: temporary - create testid user */
```

Comment:
Why we are removing this code section  .. I think this is default user getting created for Rados backend as well. We should be calling "user->store_user(dpp, null_yield, true);" to store this default user in Motr

Reply:
As per discussion in the call.